src/api/app/views/webui/package/update_build_log.js.erb

```diff
@@ -1,5 +1,7 @@
 <% if @finished %>
+  $('#log_space').empty();
   $('#log_space').append('<%= escape_javascript(@log_chunk) %>');
```

Comment:
You can use [html(string)](http://api.jquery.com/html/#html2) to replace the html of an element.